pkg/scheduler/framework/plugins/nodeports/node_ports.go

```diff
@@ -61,9 +61,8 @@ func (pl *NodePorts) Name() string {
 func getContainerPorts(pods ...*v1.Pod) []*v1.ContainerPort {
 	ports := []*v1.ContainerPort{}
 	for _, pod := range pods {
-		for j := range pod.Spec.Containers {
-			container := &pod.Spec.Containers[j]
-			for k := range container.Ports {
+		for _, container := range pod.Spec.Containers {
```

Comment:
Although it's basically stylish difference, I still prefer the old style as the new style would generate a copy of `Container` (unlike pods, pods is a slice with Pod pointer) each time, and `Container` has a ton of fields inside.